e2e/jest.config.js

```diff
@@ -3,7 +3,6 @@ const { TEST_MODE } = process.env;
 
 module.exports = {
   "verbose": false,
-  "bail": 1,  // Stop running tests after `n` failures
```

Comment:
setting `bail` prevents test log write to file. Alternative is implemented in `PuppeteerCustomEnvironment`'s `handleTestEvent`.